firstvoices/backend/models/song.py

```diff
@@ -41,17 +48,11 @@ class Meta:
             "delete": predicates.can_delete_controlled_data,
         }
 
-    title = models.CharField(blank=False, null=False)
-    title_translation = models.CharField(blank=True, null=False)
-
-    introduction = models.CharField(blank=True, null=False)
-    introduction_translation = models.CharField(blank=True, null=False)
-
     acknowledgements = ArrayField(
-        models.TextField(max_length=NOTE_MAX_LENGTH), blank=True, default=list
+        models.CharField(max_length=MAX_NOTE_LENGTH), blank=True, default=list
     )
     notes = ArrayField(
-        models.TextField(max_length=NOTE_MAX_LENGTH), blank=True, default=list
+        models.CharField(max_length=MAX_NOTE_LENGTH), blank=True, default=list
```

Comment:
ArrayFields don't do well in mixins because of whatever magic they do under the hood, so I left these alone for now. Whenever we decide to standardize all our note and ack fields, we can come up with some reusable method between songs, stories, story pages, dictionary entries, etc